### PR TITLE
Fix shunt elements 3ph calculation

### PR DESCRIPTION
**Pull request by nitbharambe**
## Problem Description

This PR fixes the issue https://github.com/e2nIEE/pandapower/issues/1892

## Solution Description

A guess is that the `ppc` seems to be incorrectly constructed for shunt elements like `shunt`, `ward`, `xward`.

When looking at `build_branch.py`, we see a mention of `pf_3ph` modifying the `baseR`.
https://github.com/e2nIEE/pandapower/blob/bca4efe14d9eea5338c792a9e10bf51dd084047f/pandapower/build_branch.py#L145-L152

Comparing this to `build_bus.py`, a mention of `pf_3ph` mode seems to not be available:

https://github.com/e2nIEE/pandapower/blob/bca4efe14d9eea5338c792a9e10bf51dd084047f/pandapower/build_bus.py#L459-L472

The modifications does give out desired results where the power flow calculation matches for minimal example in the issue.

Does this solution make sense?
 
Thanks for your time! :)

### CODE REVIEW

**Comment by rbolgaryn** [comment]
Dear @nitbharambe ,

we are taking a closer look right now. So far, these shunt elements had not been adjusted for 3ph calculation.

Can you please add some tests that cover your changes? Such tests can be small minimal examples that isolate the element and assert the results are matching the reference values.

Also, please add a mention in the CHANGELOG file.

**Comment by rbolgaryn** [comment]
Dear @nitbharambe ,

please add the information in `CHANGELOG.rst` and add a test in `pandapower/test/loadflow/test_runpp_3ph.py`

Then we can merge this pull request.

Thank you for your contribution to pandapower!

**Comment by nitbharambe** [comment]
> Dear @nitbharambe ,
> 
> please add the information in `CHANGELOG.rst` and add a test in `pandapower/test/loadflow/test_runpp_3ph.py`
> 
> Then we can merge this pull request.
> 
> Thank you for your contribution to pandapower!

Apologies for the delay @rbolgaryn, I have been travelling recently and I missed to address the comments earlier. Added the test and changelog entry.

**Comment by nitbharambe** [comment]
I can however see the `docs_check` failing for this PR in previous commits and I haven't made any modifications to documentations. Let me know if there is anything more required to be done.

**Comment by rbolgaryn** [comment]
Thank you @nitbharambe !

The docs check was added quite recently and we haven't made it to pass yet, this is still our internal todo.

File: pandapower/test/loadflow/test_runpp_3ph.py [hunk 1]
```diff
@@ -532,6 +532,34 @@ def test_3ph_with_impedance():
...
+                 "q_a_to_mvar", "p_b_to_mw", "q_b_to_mvar", "p_c_to_mw", "q_c_to_mvar"]
+    line_power_pp = np.abs(net.res_line_3ph[line_cols].values)
+    line_power_expected = np.abs(np.array(
+        [[0.33167495789351165, 0.33163327786948577, 0.33167495789351165, 0.33163327786948565, 0.33163327786948565,
+          -0.3300249368894127, -0.3300249368948133, -0.3300249368894127, -0.3300249368948133, -0.3300249368894127,
+          -0.3300249368948134]]))
+    assert np.max(np.abs(line_power_pp - line_power_expected)) < 1e-5
+
+
 if __name__ == "__main__":
```
Comment (rbolgaryn) [inline pandapower/test/loadflow/test_runpp_3ph.py:563]: I just noticed this here: please change this line here to `pytest.main([__file__])`

Comment (rbolgaryn) [reply pandapower/test/loadflow/test_runpp_3ph.py:563]: @nitbharambe

Comment (nitbharambe) [reply pandapower/test/loadflow/test_runpp_3ph.py:563]: Changed

**Review by rbolgaryn — DISMISSED** [review]

**Review by rbolgaryn — APPROVED** [review]

**Pull request merged**